es8/2017-03/mar-21.md

```diff
@@ -0,0 +1,1467 @@
+
+BT: How do you see substr() as different from anything else in Annex B.
+
+AWB: Some of the other functions are in there because they have limited applicability or Y2K issues or some other bad aspect to them. I don't see anything that bad about substr(). The concern about creating confusion doesn't apply. It is what it is; implemented in all engines, etc.
+
+BT: What's the concern?
+
+AWB: Because none of the Annex B criteria really apply to it.
+
+YK: What are the criteria.
+
+AWB: Things that have issues with them.
+
+YK: I think that we should merge Annex B into the spec. There's a strong sentiment among people (including implementers) on this point and I think we should have someone come with a prepared presentation on this subject.
+
+BT: Step 1 is inline it.
+
+YK: The answer is not "it is normative".
+
+BT: It is not normative. It is in the clause that...
+
+AWB: I was having a discussion with someone about Node using normal sloppy mode using normal function semantics. And I asked "Why is that?" And it's because that's what V8 does. Because you should never actually see that.
+
+YK: There's a lot of JavaScript in the world and implementers all need to implement Annex B.
+
+WH: That's throwing in the towel. Don't want to make Annex B normative.
+
+AWB: In the web you have lots of code that nobody can maintain. That's not the case on Node.js.
+
+AK: In the interest of moving along, I think it'd be good for us to end this conversation for now and push it on to another time.
+
+YK: Can I say one more thing? I think that if people want to advocate for not throwing in the towel, then we should come up with the goals for this.
+
+AWB: This would require evangelizing implementations to say "you would have to switch".
+
+MM: That's not required for my goals. I wouldn't mind a switch, but none of the main implementations needs to have a switch in order to meet my goals.
+
+AWB: That would be required if you say these things have to switch.
+
+MM: None of this would be required with the goals I have in mind.
+
+BT: I'm coming at it from an implementer's standpoint.
+
+MM: We should just have a face-to-face conversation Brian to talk it out.
+
+BT: Ok.
+
+SYG: I have been told Caridy is on the call now.
+
+#### Conclusion/Resolution
+
+- Further discussion on Annex B
+
+
+## 7. ECMA-402 status update
+
+(Caridy Patiño)
+
+AWB: We already have the vote to approve the final draft. Hopefully you will tell us that it's ready to be approved. _chuckling_
+
+DFV: Having issues hearing him. He's joining now.
+
+DH: I am pinging him now. Some issues with audio setup. Is that Diego?
+
+DFV: ...
+
+DH: Our sound person says you are showing up as self-muted.
+
+CP: ...
+
+DFV: I'd say you should move on and try again.
+
+AWB:  We'll go ahead and move on to the 15-minute items.
+
+## 10.i.a. Template Literal Updates
```

Comment:
The sections would benefit from linking to the proposals.  For example, I glean this section is about https://tc39.github.io/proposal-template-literal-revision/ but it's hard to say for sure.